src/main/java/org/jenkinsci/test/acceptance/po/SlavesMixIn.java

```diff
@@ -54,4 +54,12 @@ private String remoteFs(String name) {
 
         return base + name;
     }
+
+    /**
+     * Get the master node of jenkins.
+     * @return The master node.
+     */
+    public DumbSlave getMaster() {
```

Comment:
Note that master *is not* a `DumbSlave`. All shared logic between master and slaves should be in `Node` class which `Jenkins` implements so it should be ok for you to talk to `Jenkins` directly. You might need to push some behavior up the class hierarchy from `DumbSlave` to `Node`, though.